src/benchmark/file_io_read_mmap_benchmark.cpp

```diff
@@ -155,17 +155,17 @@ BENCHMARK_DEFINE_F(FileIOMicroReadBenchmarkFixture, MMAP_ATOMIC_MAP_SHARED_SEQUE
 }
 
 BENCHMARK_REGISTER_F(FileIOMicroReadBenchmarkFixture, MMAP_ATOMIC_MAP_PRIVATE_SEQUENTIAL)
-    ->ArgsProduct({{100}, {1, 2, 4, 8, 16, 32}})
+    ->ArgsProduct({{100}, {1, 2, 4, 8, 16, 24, 32, 48}})
     ->UseRealTime();
 BENCHMARK_REGISTER_F(FileIOMicroReadBenchmarkFixture, MMAP_ATOMIC_MAP_PRIVATE_RANDOM)
-    ->ArgsProduct({{100}, {1, 2, 4, 8, 16, 32}})
+    ->ArgsProduct({{100}, {1, 2, 4, 8, 16, 24, 32, 48}})
     ->UseRealTime();
 
 BENCHMARK_REGISTER_F(FileIOMicroReadBenchmarkFixture, MMAP_ATOMIC_MAP_SHARED_SEQUENTIAL)
-    ->ArgsProduct({{100}, {1, 2, 4, 8, 16, 32}})
+    ->ArgsProduct({{100}, {1, 2, 4, 8, 16, 24, 32, 48}})
     ->UseRealTime();
 BENCHMARK_REGISTER_F(FileIOMicroReadBenchmarkFixture, MMAP_ATOMIC_MAP_SHARED_RANDOM)
-    ->ArgsProduct({{100}, {1, 2, 4, 8, 16, 32}})
+    ->ArgsProduct({{100}, {1, 2, 4, 8, 16, 24, 32, 48}})
```

Comment:
IÄm not sure why filesize here is only 100, but could you change it back to 1, 10 and 100?